src/LoginRouter.js

```diff
@@ -135,6 +137,8 @@ export default BaseLoginRouter.extend({
     'signin/consent': 'consentRequired',
     'signin/admin-consent': 'adminConsentRequired',
     'signin/enroll-user': 'enrollUser',
+    'signin/device-activate-complete': 'deviceActivateComplete',
+    'activate': 'deviceActivate',
```

Comment:
how about `/signin/device/activate`?

Reply:
I would rather do `/signin/device-activate` to keep it consistent with other url patterns here. One thing to note though, the flow is going to start with `/activate` and then redirect to `/signin/device-activate`. @larsjohansen-okta  we good with that?

Reply:
> the flow is going to start with /activate and then redirect to /signin/device-activate.

why need this redirect instead of going to `signin/device-activate` directly?

Reply:
PM wants starting url to be `/activate` since its going to be entered by users on their phones so it needs to be easy to type in. So the flow is going to be 

1. user goes to foo.okta.com/activate
2. backend sets the state token and return login page
3. siw calls /introspect
4. /introspect returns `DEVICE_ACTIVATE` status which triggers route change to `signin/device-activate` and renders user code form. 

cc: @larsjohansen-okta